components/sendgrid/actions/add-email-to-global-supression/add-email-to-global-supression.js

```diff
@@ -0,0 +1,23 @@
+const common = require("../common");
+
+module.exports = {
+  ...common,
+  key: "sendgrid-add-email-to-global-supression",
+  name: "Add Email To Global Supression",
+  description:
+    "Allows you to add one or more email addresses to the global suppressions group.",
```

Comment:
Can you also point out in the `description` property the api endpoint docs on each action/source?  something like:
```suggestion
  description:
    "Allows you to add one or more email addresses to the global suppressions group. [See the docs here](https://sendgrid.api-docs.io/v3.0/suppressions-global-suppressions/add-recipient-addresses-to-the-global-suppression-group)",
```